### ISSUE TITLE
slim change the learning rate

### ISSUE THREAD

**Issue by neouyghur** [tensorflow/models#1973]
Hi, I am using slim retrain the inception network. I am using the adam optimizer. The problem is that the loss decreases very slowly. I think it is because of low learning rate. Can I change the learning rate by stopping the program and rerunning it?

**Comment by drpngx** [comment]
Yes, you should be. The learning rate is set at [construction](https://www.github.com/tensorflow/tensorflow/blob/master/tensorflow/python/training/adam.py#L92) and should be propagated throughout.

**Comment by neouyghur** [comment]
@drpngx  what is the current learning rate?  where is it saved?

**Comment by drpngx** [comment]
It's the one that you use in the python file. The current learning is computed from the original learning rate and the steps: https://www.github.com/tensorflow/tensorflow/blob/master/tensorflow/python/training/adam.py#L55

**Comment by neouyghur** [comment]
My program is now running, say it is learning rate is B. The initial
learning rate is A. If I stop the program here, and continue it with the
latest check point. The learning rate will be A or a value close to B?
